atlasdb-ete-tests/src/main/java/com/palantir/atlasdb/AtlasDbEteServer.java

```diff
@@ -77,18 +79,22 @@ public void run(AtlasDbEteConfiguration config, final Environment environment) t
     private TransactionManager tryToCreateTransactionManager(AtlasDbEteConfiguration config, Environment environment)
             throws InterruptedException {
         if (config.getAtlasDbConfig().initializeAsync()) {
-            return createTransactionManager(config.getAtlasDbConfig(), environment);
+            return createTransactionManager(config.getAtlasDbConfig(), environment, config.getAtlasDbRuntimeConfig());
         } else {
-            return createTransactionManagerWithRetry(config.getAtlasDbConfig(), environment);
+            return createTransactionManagerWithRetry(config.getAtlasDbConfig(), config.getAtlasDbRuntimeConfig(),
+                    environment
+            );
```

Comment:
nit: The ordering on the factories should be consistent (i.e. install/runtime/environment, or install/environment/runtime). I generally prefer install/runtime/environment as this is closer to internal server framework's ordering

Reply:
The ordering currently is install/runtime/environment.

Reply:
Not on `createTransactionManager`.